cpu/sam0_common/periph/rtc_rtt.c

```diff
@@ -374,6 +384,26 @@ void rtc_tamper_enable(void)
     DEBUG("tamper enabled\n");
 }
 
+uint8_t rtc_get_tamper_event(void)
+{
+    uint32_t ret = RTC->MODE0.TAMPID.reg;
+
+    /* clear tamper event */
+    RTC->MODE0.INTFLAG.reg = RTC_MODE0_INTFLAG_TAMPER;
+    RTC->MODE0.TAMPID.reg  = ret;
+
+    return ret & 0x1F;
```

Comment:
Why 0x1f?

Reply:
You're right, `RTC_TAMPID_TAMPID_Msk` is the better choice here.